main.py

```diff
@@ -0,0 +1,223 @@
+                lowest["temperature"] = int(record['lowest_temperature'])
+                lowest['date'] = record['date']
+            if record['max_humidity'].isnumeric() and int(record['max_humidity']) > humidest['temperature']:
+                humidest['temperature'] = int(record['max_humidity'])
+                humidest['date'] = record['date']
+    highest_date = highest['date'].split('-')
+    lowest_date = lowest['date'].split('-')
+    humidest_date = humidest['date'].split('-')
+    print(
+        f"Highest: {highest['temperature']}C on {datetime.datetime(int(highest_date[0]), int(highest_date[1]), int(highest_date[2])).strftime('%B %d')}")
+    print(
+        f"Lowest: {lowest['temperature']}C on {datetime.datetime(int(lowest_date[0]), int(lowest_date[1]), int(lowest_date[2])).strftime('%B %d')}")
+    print(
+        f"Humidity: {humidest['temperature']}% on {datetime.datetime(int(humidest_date[0]), int(humidest_date[1]), int(humidest_date[2])).strftime('%B %d')}")
+
+
+def check_command_month(data):
+    data = data.split('/')
+    if int(data[1]) > max(MONTHS.keys()) or int(data[1]) < min(MONTHS.keys()):
+        return -1
+    elif MONTHS[int(data[1])]:
+        return MONTHS[int(data[1])]
+    else:
+        return -1
+
+
+def check_month_in_command(command_data):
+    data = command_data.split('/')
+    if len(data) == 1:
+        return False
+    else:
+        return True
+
+
+def get_month_data_from_data(data, month):
+    return next((item for item in data if item["month"] == month), None)
+
+
+def print_monthly_average_data(data, command_data):
+    year, month = get_year_month_from_command(command_data)
+    highest_temp_data = {
+        "sum": 0,
+        "count": 0,
+    }
+    lowest_temp_data = {
+        "sum": 0,
+        "count": 0,
+    }
+    humidity_temp_data = {
+        "sum": 0,
+        "count": 0
+    }
+    for record in data['records']:
+        if record['highest_temperature'] != '':
+            highest_temp_data['sum'] += int(record['highest_temperature'])
+            highest_temp_data["count"] += 1
+        if record['lowest_temperature'] != '':
+            lowest_temp_data["sum"] += int(record['lowest_temperature'])
+            lowest_temp_data["count"] += 1
+        if record['mean_humidity'] != '':
+            humidity_temp_data["sum"] += int(record['mean_humidity'])
+            humidity_temp_data["count"] += 1
+
+    highest_average = highest_temp_data["sum"] // highest_temp_data["count"]
+    lowest_average = lowest_temp_data["sum"] // lowest_temp_data["count"]
+    average_humidity = humidity_temp_data["sum"] // humidity_temp_data["count"]
+
+    print(f'Highest Average: {highest_average}')
+    print(f'Lowest Average: {lowest_average}')
+    print(f'Average Mean Humidity: {average_humidity}')
+
+
+def draw_month_graph(data, month):
+    month_data = get_month_data_from_data(data, month)
+    print(month_data)
```

Comment:
What is the point of storing in month_data if we are just going to display it? We can directly use print statement.